### PR TITLE
Document external registry usage

### PR DESCRIPTION
**Pull request by lkolacek**
CLOUDBLD-1884

Signed-off-by: Ladislav Kolacek <lkolacek@redhat.com>

### CODE REVIEW

File: docs/users.rst [hunk 1]
```diff
@@ -953,6 +953,47 @@ image builds are not enabled (see :ref:`group_manifests
 <group-manifests>`), to the sole image manifest resulting from the
 build.
 
+Parent Image in registry.redhat.io
```
Comment (MartinBasti) [inline docs/users.rst:956]: This should be generic external registry, it depends on used configuration

File: docs/users.rst [hunk 2]
```diff
@@ -953,6 +953,47 @@ image builds are not enabled (see :ref:`group_manifests
...
+----------------------------------
+
+OSBS uses the ``FROM`` instruction in the Dockerfile to find a parent image
+build. When the user does not specify the source registry, OSBS will
+automatically add the internal registry ``Quay.io``.
```
Comment (MartinBasti) [inline docs/users.rst:961]: configuration detail, call it "OSBS source registry"

**Comment by MartinBasti** [comment]
NACK, this is RH only specific. OSBS upstream documentation should contain generic howto, because all of this is configurable

**Comment by lkolacek** [comment]
Martin pointed me to right place for this documentation. Closing

**Pull request closed without merging**